tyk-headless/values.yaml

```
@@ -24,34 +24,48 @@ secrets:
   useSecretName: ""
 
 redis:
-  # The addrs value will allow you to set your Redis addresses. If you are
-  # using a redis cluster, you can list the endpoints of the redis instances
-  # or use the cluster configuration endpoint.
+  # The addrs value will allow you to set your Redis addresses.
+  #
+  # If you are using Redis (e.g. Bitnami Redis at bitnami/redis) then enter single
+  # endpoint. If using sentinel connection mode for Redis, please update the port number (typically 26379).
+  #
+  # If using a Redis Cluster (e.g. bitnami/redis-cluster), you can list
+  # the endpoints of the redis instances or use the cluster configuration endpoint.
+  #
   # Default value: redis.{{ .Release.Namespace }}.svc.cluster.local:6379
   # addrs:
+  #   Example using tyk simple redis chart
   #   - redis.tyk.svc.cluster.local:6379
-  #   This is the the DNS name of the redis as set when using Bitnami
+  #   Example using bitnami/redis
+  #   - tyk-redis.tyk.svc.cluster.local:6379
```

```suggestion
  #   - tyk-redis-master.tyk.svc.cluster.local:6379
```
For bitnami redis, we need to connect to `tyk-redis-master.tyk.svc.cluster.local:6379` address